packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```
@@ -44,6 +41,9 @@ BuildRequires: go
 %else
 BuildRequires: golang
 %endif
+%if 0%{?rhel} > 8 || 0%{?fedora}
```

I'm not a fan of `>`. Perhaps it's my mind, but I always find this easier so I don't need to do the + 1 in my head
```suggestion
%if 0%{?rhel} >= 9 || 0%{?fedora}
```